server/src/main/java/org/elasticsearch/search/rank/rrf/RRFRankResult.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.search.rank.rrf;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.search.rank.RankResult;
+import org.elasticsearch.xcontent.XContentBuilder;
+
+import java.io.IOException;
+
+public class RRFRankResult extends RankResult {
+
+    public static final String NAME = "rrf";
+
+    public int rank;
+    public final int[] positions;
+    public final float[] scores;
+
+    public RRFRankResult(int doc, int shardIndex, int size) {
```

Comment:
Would be neat if we knew the the position group. 

Also, what about when a doc is in one set, but not another?

Reply:
I need to add some documentation here for sure. `positions` is always the size of the number of queries executed (excluding the aggs query). The positions are 1...n, so any document that isn't part of a query's result set is just naturally 0. Maybe -1 would be better to be more explicit?

Reply:
This is the same for now.